asdf/tests/test_generic_io.py

```diff
@@ -303,7 +303,7 @@ def get_read_fd():
         if len(tree) == 4:
             assert connection[0]._nreads == 0
         else:
-            assert connection[0]._nreads == 6
+            assert connection[0]._nreads == 5
```

Comment:
I don't think this change is necessary. In any case it causes failures on Travis Ci.